### PR TITLE
UI - Fix intl.translate in react components

### PR DESCRIPTION
**Pull request by carlito-scality**
**Component**:UI

**Context**: 

- Once IntlGlobalProvider is introduced to allow us to translate text in sagas, intl.translate does not work anymore in React components, only intl.messages works. It triggers an JS error 'translate is not a function'. We need to use ONLY IntlGlobalProvider's intl.translate everywhere (saga and react components) for translation to be consistent. Thus we can get rid of HOC injectIntl

**Summary**:

**Acceptance criteria**: 

---

Closes: https://github.com/scality/metalk8s/issues/2122

### CODE REVIEW

File: ui/src/translations/IntlGlobalProvider.js [hunk 1]
```diff
@@ -36,10 +36,13 @@ class IntlTranslator {
   // Formatting Functions
   // ------------------------------------
   translate(key, values) {
+    if (!key) {
+      return '';
+    }
```
Comment (slaperche-scality) [inline ui/src/translations/IntlGlobalProvider.js:41]: this is the only relevant change, the rest of the diff is linting/cosmetics and belong to a dedicated commit

**Review by ChengYanJin — APPROVED** [review]
LGTM!
